src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -5483,4 +5489,82 @@ This option should be used when specifying the certificate via -CertificateSubje
     <value>When creating a symbols package, allows to choose between the 'snupkg' and 'symbols.nupkg' format.</value>
     <comment>Do not localize snupkg and symbols.nupkg</comment>
   </data>
+  <data name="PackageCommandPackagesDirectory_csy" xml:space="preserve">
```

Comment:
remove all these non-English resources. Those will be taken care in separate resource files.